src/Encode/AlphaEncodingManager.cpp

```diff
@@ -204,11 +206,18 @@ VRInputData AlphaEncodingManager::Decode(const std::string& input) {
   if (inputMap.find(VRCommDataAlphaEncodingKey::JoyY) != inputMap.end())
     joyY = 2 * std::stof(inputMap.at(VRCommDataAlphaEncodingKey::JoyY)) / maxAnalogValue_ - 1;
 
+  float trgValue = 0; 
+
+  // trigger value is 0.0f -> 1.0f inclusive
+  if (inputMap.find(VRCommDataAlphaEncodingKey::TrgValue) != inputMap.end())
+    trgValue = std::stof(inputMap.at(VRCommDataAlphaEncodingKey::TrgValue)) / maxAnalogValue_;
+
   VRInputData inputData(
       jointFlexion,
       splay,
       joyX,
       joyY,
+      trgValue,
```

Comment:
Will adding this to legacy break functionality for any gloves that do not send a trigger value input? If so, then we should not add this to legacy and copy the value of index finger for trgvalue in legacy encoding.

Reply:
Sure, I guess we shouldn't be supporting new features in legacy.